src/OpenLoco/Economy/Economy.cpp

```diff
@@ -71,7 +71,7 @@ namespace OpenLoco::Economy
                 continue;
             }
 
-            for (uint16_t numDays = 2; numDays <= 122; ++numDays)
+            for (uint16_t numDays = 2; numDays < 122; ++numDays)
```

Comment:
Is this even correct the rest of the function? `_deliveredCargoPayment[cargoItem][(numDays / 2) - 1]` shouldn't this only be called for every other day? i.e.:
```
for (uint16_t numDays = 2; numDays < 122; numDays+=2)
```

Reply:
I think you're right, yes. Would save a few cycles, but thankfully that wouldn't have caused any harm, I think?

Shall we go with this, then?
```suggestion
            for (uint16_t numDays = 2; numDays < 122; numDays += 2)
```

Reply:
See what the replays say. I'll check it tomorrow